app/assets/stylesheets/pages/_machines.scss

```diff
@@ -0,0 +1,270 @@
+.devices {
+  min-height: 60vh;
+
+  b {
+    color: $c-pink;
+    font-weight: 600;
+  }
+}
+
+.device__selection {
+  img {
+    align-self: center;
+    height: auto;
+    max-width: 135px;
+    min-width: 52px;
+    width: 20%;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  li {
+    background-color: #f5f5f5;
+    cursor: pointer;
+    display: flex;
+    flex-basis: 33.333%;
```

Comment:
you might want to also add:
```
flex-grow: 0;
flex-shrink: 0;
```
That will keep the button from trying to flex, since you want this to stick to 1/3